qtawesome/iconic_font.py

```diff
@@ -270,14 +312,23 @@ def hook(obj):
         # Load font
         if QApplication.instance() is not None:
             with open(os.path.join(directory, ttf_filename), 'rb') as font_data:
-                id_ = QFontDatabase.addApplicationFontFromData(QByteArray(font_data.read()))
+                data = font_data.read()
+                id_ = QFontDatabase.addApplicationFontFromData(data)
+                # QRawFont() requires pixelSize, but it is unknown at this time.
+                # So, first create QRowfont with 2048, which is the size of
+                # almost TrueType fonts and reset actual size after that.
+                # Note: When TrueType fonts, pixelSize can be specified up to 16384.
```

Comment:
A little improvement to the comment here:
```suggestion
                # `QRawFont()` requires a `pixelSize` value, but it is unknown at this point.
                # To workaround that, first we create a `QRawFont` with 2048 as `pixelSize` 
                # (usual size for TrueType fonts) and reset to the actual size after that.
                # Note: For TrueType fonts, `pixelSize` can be specified up to 16384.
```